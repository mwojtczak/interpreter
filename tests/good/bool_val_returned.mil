function is_positive(val i) {
	if (i > 0) {
		return(true);
	} else {
		return(false);
	}
}

function main () {
	x = 5;
	print(x);
	y = is_positive(x);
	print(y);
	x = -4;
	print(x);
	y = is_positive(x);
	print(y);
}