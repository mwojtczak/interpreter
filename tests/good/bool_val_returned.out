5
true
-4
false
